### PR TITLE
Add redhat-et/microshift 

### PR DESCRIPTION
**Pull request by copejon**
Signed-off-by: jcope <jcope@redhat.com>

### CODE REVIEW

File: ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
+zz_generated_metadata:
+  branch: main
+  org: redhat-et
+  repo: microshift
+images:
+  dockerfile_literal: |
```
Comment (sallyom) [inline ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml:30]: this needs the dash for the images list
```bash
images:
- dockerfile_literal: |
    FROM registry.access.redhat.com/ubi8/go-toolset
```
hopefully that's it

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -7706,6 +7733,34 @@ plugins:
...
+    - retitle
+    - shrug
+    - sigmention
+    - skip
+    - trigger
+    - verify-owners
+    - owners-label
+    - wip
+    - yuks
+    - approve
```
Comment (sallyom) [inline core-services/prow/02_config/_plugins.yaml:7763]: also, u can remove most of these (I'd definitely keep the cat, dog and goose :) https://prow.k8s.io/plugins
these control GH comment triggers, you'll probably want 
- lgtm
- wip
- approve (?)

**Comment by copejon** [comment]
@stevekuznetsov @petr ready for review

**Comment by stevekuznetsov** [comment]
/lgtm
/approve
:shipit:

**Pull request merged**